src/module/utils/TwodsixShipActions.ts

```diff
@@ -82,13 +89,35 @@ export class TwodsixShipActions {
 
       /*get characteristic key, default to skill key if none specificed in formula */
       let characteristicKey = "";
+      const charObject = selectedActor?.system["characteristics"] ?? {};
+      //we need an array
+      const charObjectArray = Object.keys(charObject).map(key => {
+        return charObject[key];
+      });
```

Comment:
Can do all of this in one line using Object.values
const charObjectArray = Object.values(selectedActor?.system["characteristics"]);